src/UglyToad.PdfPig/Writer/PdfPageBuilder.cs

```diff
@@ -509,13 +534,12 @@ public AddedImage AddPng(Stream pngStream, PdfRectangle placementRectangle)
         /// <param name="srcPage">Page to be copied</param>
         public void CopyFrom(Page srcPage)
```

Comment:
I guess it is too late, but it feels like this api should use the new logic. I'm afraid one might end up outdated pretty quickly

Reply:
I agree, I didn't want to change much there as it was just added by @InusualZ . It can serve a different purpose (allows pages to be overlayed on each other, allows modification of graphics operations after import) but would be nice to reduce some of the duplication.